### ISSUE TITLE
🐛 Bug Report: Bad contrast on required item numbers

### ISSUE THREAD

**Issue by Lenni009** [AssistantNMS/App#190]
### 📜 Describe the bug

Item number text color in recipes doesn't match accessibility standards.

### 👟 To Reproduce

1. Go to "Activated Cadmium"
2. See "Maximum Stack" being very readable (black on cyan)
3. See the numbers next to the icons below being not readable (white on cyan)

### 👍 Expected behavior

All text on cyan should be black for better accessibility.

### 📱 Where did you find this problem?

Android

### 👀 Have you checked if this bug report has been created already?

- [X] I checked and couldn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/AssistantNMS/App/blob/main/.github/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

None
